src/libsync/utility_win.cpp

```diff
@@ -28,8 +28,20 @@ namespace OCC {
 
 static void setupFavLink_private(const QString &folder)
 {
-    // Windows Explorer: Place under "Favorites" (Links)
+    // First create a Desktop.ini so that the folder and favorite link show our application's icon.
+    QFile desktopIni(folder + QLatin1String("/Desktop.ini"));
+    if (desktopIni.exists()) {
+        qCWarning(lcUtility) << desktopIni.fileName() << "already exists, not overwriting it to set the folder icon.";
+    } else {
+        qCInfo(lcUtility) << "Creating" << desktopIni.fileName() << "to set a folder icon in Explorer.";
+        desktopIni.open(QFile::WriteOnly);
+        desktopIni.write("[.ShellClassInfo]\r\nIconResource=");
+        desktopIni.write(qApp->applicationFilePath().toUtf8().replace('/', '\\'));
```

Comment:
Ah yes, forgot about toNativeSeparators again, I'll fix it.
I thought I saw a not on some documentation that this should be UTF-8, but I can't find it anymore. If there is non-latin1 characters in the path, it won't work if I use toLatin1 either, so I'd keep it that way.